tests/python/pants_test/engine/test_fs.py

```diff
@@ -357,6 +357,23 @@ def test_merge_directories(self):
 
       self.assertEquals(both_snapshot.directory_digest, both_merged)
 
+  def test_materialize_directories(self):
+    with temporary_dir() as temp_dir:
+      dir_path = os.path.join(temp_dir, "roland")
```

Comment:
Either call this `containing_roland` rather than `roland`, or just use `temp_dir` as the containing dir